tick/base/array/swig/array_test_cfunc.i

```diff
@@ -120,26 +120,32 @@
         *n_rows = PyLong_AsLong(obj_nrows);
         *n_cols = PyLong_AsLong(obj_ncols);
 
+#ifndef _WIN32
         if (PyArray_TYPE(obj_data) != NP_TYPE || PyArray_ITEMSIZE(obj_data) != sizeof(C_TYPE))
         {
             PyErr_SetString(PyExc_ValueError,"Expecting a " #C_TYPE " numpy array for data field of sparse matrix");
             Py_DECREF(obj_indptr);Py_DECREF(obj_indices);Py_DECREF(obj_data);Py_DECREF(obj_shape);
             return(false);
         }
+#endif        
         *data = (C_TYPE *) PyArray_DATA(obj_data);
 
+#ifndef _WIN32
         if (PyArray_TYPE(obj_indices) != NPY_INT32 || PyArray_ITEMSIZE(obj_indices) != sizeof(INDICE_TYPE)) {
             PyErr_SetString(PyExc_ValueError,"Expecting 4 bytes integer array for field indices of sparse matrix");
             Py_DECREF(obj_indptr);Py_DECREF(obj_indices);Py_DECREF(obj_data);Py_DECREF(obj_shape);
             return(false);
         }
+#endif
         *indices = (INDICE_TYPE *) PyArray_DATA(obj_indices);
 
+#ifndef _WIN32
        if (PyArray_TYPE(obj_indptr) != NPY_INT32 || PyArray_ITEMSIZE(obj_indptr) != sizeof(INDICE_TYPE)) {
             PyErr_SetString(PyExc_ValueError,"Expecting 4 bytes integer array for field indptr of sparse matrix");
             Py_DECREF(obj_indptr);Py_DECREF(obj_indices);Py_DECREF(obj_data);Py_DECREF(obj_shape);
             return(false);
         }
+#endif
```

Comment:
Let's create an issue to figure out when this is necessary

Reply:
sounds good

Reply:
done https://github.com/X-DataInitiative/tick/issues/100